public/app/features/dashboard/partials/dashboardTopNav.html

```diff
@@ -29,14 +29,14 @@
 				<li ng-show="dashboardMeta.canSave">
 					<a ng-click="saveDashboard()" bs-tooltip="'Save dashboard'" data-placement="bottom"><i class="fa fa-save"></i></a>
 				</li>
-				<li class="dropdown">
+				<li ng-if="dashboardMeta.canEdit" class="dropdown">
```

Comment:
This makes it impossible to make a dashboard editable again if its none editable (Settings -> Toggles -> Editable ) even for admins.